### PR TITLE
Update migrate-prepare-intune.md

### PR DESCRIPTION
**Pull request by saurabhkoshta**
### Summarize the change in the pull request title

Added additional information under "Notes" to reflect user side behavior.

### CODE REVIEW

**Comment by PRMerger7** [comment]
@saurabhkoshta : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

File: sccm/mdm/deploy-use/migrate-prepare-intune.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ If you [imported Configuration Manager data to Microsoft Intune](migrate-import-
 - [Assign policies](https://docs.microsoft.com/intune/get-started-policies)  
 - [Assign profiles](https://docs.microsoft.com/intune/device-profile-assign)  
     > [!NOTE]  
-    > When Intune deploys the new email profile, users receive a prompt to reenter their password.  
+    > When Intune deploys the new email profile, users receive a prompt to reenter their password. This will result in emails being re-downloaded on users devices and any custom modifications done by the user will needed to be done again.
```
Comment (aczechowski) [inline sccm/mdm/deploy-use/migrate-prepare-intune.md]: A few editorial comments:

```suggestion
    > When Intune deploys the new email profile, users receive a prompt to reenter their password. This behavior results in emails being redownloaded on users' devices. Any custom modifications done by the user will need to be done again. 
```

**Comment by PRMerger15** [comment]
Microsoft authors of articles need to make updates through the private repository so the updates can be staged and validated by the current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged.It's easy to access the private repo version of your articles in GitHub. Just add -pr to the URL, as follows: 
 Public repo URL: https://github.com/Microsoft/azure-docs/blob/master/articles/azure-portal-overview.md 
 Private repo URL: https://github.com/Microsoft/azure-docs-pr/blob/master/articles/azure-portal-overview.md 
 We have closed this pull request. Please resubmit the updates through the private repo. If you need perms to the private repo, [see the contributor guide instructions](https://github.com/Microsoft/azure-docs/blob/master/contributor-guide/tools-and-setup.md#permissions)

**Comment by aczechowski** [comment]
Automation bug (in my opinion), reopening

**Comment by aczechowski** [comment]
#sign-off 

*Once merged*, this should be live with the next daily publishing process. Right now that *should* be tomorrow (Wednesday, 16 January) around 11:00 Pacific time. No further action needed from you.

**Pull request merged**